compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/DerivedConfigUtils.cpp

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 The IREE Authors
```

Comment:
Commenting for other reviewers, since it may not be clear. The contents of this file were moved from `TargetUtils/ConfigUtils.cpp` (which has been replaced with new logic in that file).